manifest.json

```diff
@@ -36,13 +37,13 @@
   },
 
   "requirements": [
-    "MarkLogic 9+"
+    "MarkLogic 10+"
```

Comment:
Note that the sink connector should still be fine on ML 9 (and likely ML 8 too); an array is supported here, so maybe it's better to do "MarkLogic 9 for sink", "MarkLogic 10 for source" or something like that?